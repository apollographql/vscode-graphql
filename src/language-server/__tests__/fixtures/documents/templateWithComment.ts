`#graphql
query {test}`;

` # graphql
query {test}`;

`

# graphql

query {
  test
}
`;

`# GraphQL
query {
  test
}`;

`#gql
query {
  test
}`;

`#graphql
query {
  test
}`;
