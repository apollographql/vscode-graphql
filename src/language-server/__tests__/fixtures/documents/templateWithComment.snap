>`#graphql
#^ inline.graphql apollo.templateWithComment punctuation.definition.string.template.begin.js
# ^^^^^^^^ inline.graphql apollo.templateWithComment comment.line.graphql.js
>query {test}`;
#^^^^^ inline.graphql apollo.templateWithComment meta.embedded.block.graphql keyword.operation.graphql
#     ^ inline.graphql apollo.templateWithComment meta.embedded.block.graphql meta.selectionset.graphql
#      ^ inline.graphql apollo.templateWithComment meta.embedded.block.graphql meta.selectionset.graphql punctuation.operation.graphql
#       ^^^^ inline.graphql apollo.templateWithComment meta.embedded.block.graphql meta.selectionset.graphql variable.graphql
#           ^ inline.graphql apollo.templateWithComment meta.embedded.block.graphql meta.selectionset.graphql punctuation.operation.graphql
#            ^ inline.graphql apollo.templateWithComment punctuation.definition.string.template.end.js
#             ^^ inline.graphql
>
>` # graphql
#^^^^^^^^^^^^ inline.graphql
>query {test}`;
#^^^^^^^^^^^^^^^ inline.graphql
>
>`
#^^ inline.graphql
>
># graphql
#^^^^^^^^^^ inline.graphql
>
>query {
#^^^^^^^^ inline.graphql
>  test
#^^^^^^^ inline.graphql
>}
#^^ inline.graphql
>`;
#^^^ inline.graphql
>
>`# GraphQL
#^ inline.graphql apollo.templateWithComment punctuation.definition.string.template.begin.js
# ^^^^^^^^^ inline.graphql apollo.templateWithComment comment.line.graphql.js
>query {
#^^^^^ inline.graphql apollo.templateWithComment meta.embedded.block.graphql keyword.operation.graphql
#     ^ inline.graphql apollo.templateWithComment meta.embedded.block.graphql meta.selectionset.graphql
#      ^ inline.graphql apollo.templateWithComment meta.embedded.block.graphql meta.selectionset.graphql punctuation.operation.graphql
>  test
#^^ inline.graphql apollo.templateWithComment meta.embedded.block.graphql meta.selectionset.graphql
#  ^^^^ inline.graphql apollo.templateWithComment meta.embedded.block.graphql meta.selectionset.graphql variable.graphql
>}`;
#^ inline.graphql apollo.templateWithComment meta.embedded.block.graphql meta.selectionset.graphql punctuation.operation.graphql
# ^ inline.graphql apollo.templateWithComment punctuation.definition.string.template.end.js
#  ^^ inline.graphql
>
>`#gql
#^ inline.graphql apollo.templateWithComment punctuation.definition.string.template.begin.js
# ^^^^ inline.graphql apollo.templateWithComment comment.line.graphql.js
>query {
#^^^^^ inline.graphql apollo.templateWithComment meta.embedded.block.graphql keyword.operation.graphql
#     ^ inline.graphql apollo.templateWithComment meta.embedded.block.graphql meta.selectionset.graphql
#      ^ inline.graphql apollo.templateWithComment meta.embedded.block.graphql meta.selectionset.graphql punctuation.operation.graphql
>  test
#^^ inline.graphql apollo.templateWithComment meta.embedded.block.graphql meta.selectionset.graphql
#  ^^^^ inline.graphql apollo.templateWithComment meta.embedded.block.graphql meta.selectionset.graphql variable.graphql
>}`;
#^ inline.graphql apollo.templateWithComment meta.embedded.block.graphql meta.selectionset.graphql punctuation.operation.graphql
# ^ inline.graphql apollo.templateWithComment punctuation.definition.string.template.end.js
#  ^^ inline.graphql
>
>`#graphql
#^ inline.graphql apollo.templateWithComment punctuation.definition.string.template.begin.js
# ^^^^^^^^ inline.graphql apollo.templateWithComment comment.line.graphql.js
>query {
#^^^^^ inline.graphql apollo.templateWithComment meta.embedded.block.graphql keyword.operation.graphql
#     ^ inline.graphql apollo.templateWithComment meta.embedded.block.graphql meta.selectionset.graphql
#      ^ inline.graphql apollo.templateWithComment meta.embedded.block.graphql meta.selectionset.graphql punctuation.operation.graphql
>  test
#^^ inline.graphql apollo.templateWithComment meta.embedded.block.graphql meta.selectionset.graphql
#  ^^^^ inline.graphql apollo.templateWithComment meta.embedded.block.graphql meta.selectionset.graphql variable.graphql
>}`;
#^ inline.graphql apollo.templateWithComment meta.embedded.block.graphql meta.selectionset.graphql punctuation.operation.graphql
# ^ inline.graphql apollo.templateWithComment punctuation.definition.string.template.end.js
#  ^^ inline.graphql
>